src/f5_tts/model/cfm.py

```diff
@@ -98,17 +98,18 @@ def sample(
         edit_mask=None,
     ):
         self.eval()
-
-        if next(self.parameters()).dtype == torch.float16:
-            cond = cond.half()
-
         # raw wave
 
         if cond.ndim == 2:
             cond = self.mel_spec(cond)
             cond = cond.permute(0, 2, 1)
             assert cond.shape[-1] == self.num_channels
 
+        assert next(self.parameters()).dtype == torch.float32 or next(self.parameters()).dtype == torch.float16, print(
```

Comment:
not sure if bf16 ok for `vocos`
remove assertion, maybe just previous dtype passed in load_model() is fine